modules/eco-poison-pill-about-watchdog.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-watchdog-devices_{context}"]
+= About watchdog devices
+
+Watchdog devices can be any of the following:
+
+* Independently powered hardware devices 
+* Hardware devices that share power with the hosts they control
+* Virtual devices implemented in software (or `softdog`)
```

Comment:
We try to avoid the use of parenthesis, so perhaps you can use a comma before the "or" statement instead